src/Randousel/randousel.svelte

```diff
@@ -0,0 +1,99 @@
+<script>
+  import { Card, CardSubtitle, CardTitle, CardText, CardActions, Button, MaterialApp } from 'svelte-materialify';
+  import { format } from 'd3';
+
+  export let location;
+  export let wtfData;
+  let count = 0;
+  let data = []; // Initialize data
+  let selectedIndex = 0; // Randomly pick an array element
+  let selectedWTF;
+
+  const loading = Promise.all([location, wtfData])
+    .then( ([locationResponse, wtfResponse]) => {
+      data = wtfResponse['data']['requests'].filter(d => d.wtf);
+      console.log(data)
+      selectedIndex = data.findIndex(d => d.state === locationResponse.region);
```

Comment:
If you take the other suggestion then this could happen right where `let selectedIndex` is defined.